README.md

```diff
@@ -34,6 +34,12 @@ If you are a Mac OS X user, you can use [Homebrew](https://brew.sh):
 brew install terraform-docs
 ```
 
+Windows users can install using [chocolatey](https://www.chocolatey.org):
```

Comment:
Also one minor item, can you change the link to `Chocolatey` since it's a name.

Reply:
Good catch, thanks! This is complete.